libsolidity/formal/SolverInterface.h

```diff
@@ -231,8 +231,11 @@ class SolverInterface
 	/// is available. Throws SMTSolverError on error.
 	virtual std::pair<CheckResult, std::vector<std::string>>
 	check(std::vector<Expression> const& _expressionsToEvaluate) = 0;
-};
 
+protected:
+	// SMT query timeout in milliseconds.
+	static int const queryTimeout = 10000;
```

Comment:
@chriseth please check this style and comment here if you dislike it.

Reply:
Could be `c_queryTimeout` but I think it's also fine like this.